NetSDK/Services/Cache/Interfaces/ISegmentCache.cs

```diff
@@ -7,9 +8,11 @@ namespace Splitio.Services.Cache.Interfaces
 {
     public interface ISegmentCache
     {
-        void AddToSegment(string segmentName, HashSet<string> segmentKeys);
+        void RegisterSegment(string segmentName);
```

Comment:
no, but maybe it should be just in the cache implementation and not in the interface?